packages/postgres-database/src/simcore_postgres_database/models/workspaces.py

```diff
@@ -0,0 +1,76 @@
+import sqlalchemy as sa
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+
+workspaces = sa.Table(
+    "workspaces",
+    metadata,
+    sa.Column(
+        "workspace_id",
+        sa.BigInteger,
+        nullable=False,
+        autoincrement=True,
+        primary_key=True,
+        doc="Workspace index",
+    ),
+    sa.Column("name", sa.String, nullable=False, doc="Display name"),
+    sa.Column("description", sa.String, nullable=True, doc="Short description"),
+    sa.Column(
+        "owner_primary_gid",
+        sa.BigInteger,
+        sa.ForeignKey(
+            "groups.gid",
+            name="fk_workspaces_gid_groups",
+            onupdate="CASCADE",
+            ondelete="RESTRICT",
+        ),
+        nullable=False,
+        doc="Identifier of the group that owns this workspace (Should be just PRIMARY GROUP)",
+    ),
+    sa.Column(
+        "thumbnail",
+        sa.String,
+        nullable=True,
+        doc="Link to image as to workspace thumbnail",
+    ),
+    column_created_datetime(timezone=True),
+    column_modified_datetime(timezone=True),
+    sa.Column("product_name", sa.String, nullable=False, doc="Product name"),
```

Comment:
Let's try to group the columns  logically. This will simplify checking it in adminer. 
e.g. 
- The timestamps at the end, 
- the name, thumbnail, description together
-

Reply:
improved